course_content/notebooks/dask_intro.ipynb

```diff
@@ -728,13 +728,364 @@
+   "metadata": {},
+   "source": [
+    "Computing the following will have the same result as the original iris.load."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "phenom_dask_cubes = phenom_dask_load.compute()\n",
+    "phenom_dask_cubes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 2. Constraining by Coordinate Values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another way to constrain Iris loading is based dimensions and coordinate values. In this example we will only load data with a model level number less than 10. The constraint mln_cstr will do this and mln_cstr_list will have copies of that restraint which can be used in a mapping."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "mln_cstr = iris.Constraint(model_level_number=lambda cell: cell < 10)\n",
+    "mln_cstr_list = [mln_cstr] * len(files)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following cell performs a simple iris.load using the constraint we specified above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "mln_cubes = iris.load(files, mln_cstr)\n",
+    "mln_cubes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following cells a dask bag is made by mapping individual files and the model level number constraint to the delayed iris.load. You can visualize and compute this to have the same effect as the iris.load above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "mln_dask_load = db.from_delayed(map(delayed(iris.load), files, mln_cstr_list))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "mln_dask_load.visualize()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "mln_dask_cubes = mln_dask_load.compute()\n",
+    "mln_dask_cubes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3. Constraining by Attribute"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also constrain data based on its attributes. For example, in the following cells we will constrain based on the UM version being used. This is very similar to the previous example except we are using an iris.AttributeConstraint rather than an iris.Constraint object.\n",
```

Comment:
"Constrain **load**" would probably be more accurate than "constrain **data**", which is what's here currently.

Also, bits of code in back-ticks, please.